test/check-application

```diff
@@ -3086,6 +3087,19 @@ class TestApplication(testlib.MachineCase):
         b.wait_text("button.pf-v5-c-select__menu-item.pf-m-disabled", "No images found")
         b.wait_in_text(".pf-v5-c-alert.pf-m-danger", "couldn't search registry")
 
+        # Searching for full url with tag finds the image
+        b.set_input_text("#run-image-dialog-name", "tagged-busybox")
+        b.set_input_text("#create-image-image-select-typeahead", "localhost:80/my-busybox:my-tag")
+        b.click('button.pf-v5-c-toggle-group__button:contains("All")')
+        b.wait_text("button.pf-v5-c-select__menu-item:not(.pf-m-disabled)", container_name + ":my-tag")
```

Comment:
Could this also ensure that this is the only displayed image, i.e. count the number of result rows? To make sure it's not duplicated.